tools/test/examples/examples.json

```diff
@@ -333,6 +333,22 @@
         "compare_log": ["mbed-os-example-mbed-crypto/tests/getting-started.log"],
         "auto-update" : true
     },
+    {
+        "name": "mbed-os-example-psa",
+        "github":"https://github.com/ARMmbed/mbed-os-example-psa",
+        "sub-repo-example": false,
+        "subs": [],
+        "features" : [],
+        "targets" : ["ARM_MUSCA_A1", "ARM_MUSCA_B1", "K64F", "CY8CKIT_064S2_4343W"],
+        "toolchains" : ["GCC_ARM", "ARM"],
+        "exporters": [],
+        "compile" : true,
+        "export": true,
+        "test" : true,
+        "baud_rate": 115200,
```

Comment:
Is this has to be `115200`? can we use `9600`. This is a technical debt on Greentea, I but haven't got time to fix it

Reply:
OK, I'll update the example to use 9600 by default.